### PR TITLE
Feature/analytics

### PR DESCRIPTION
**Pull request by beseven**
Test webtrends with another function

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-78 in nhsuk-dev succeeded (http://gp-finder-pr-78.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-78 in nhsuk-dev succeeded (http://gp-finder-pr-78.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-78 in nhsuk-dev succeeded (http://gp-finder-pr-78.dev.c2s.nhschoices.net)

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-78 in nhsuk-dev succeeded (http://gp-finder-pr-78.dev.c2s.nhschoices.net)

File: public/js/custom-webtrends.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+jQuery(function($) {
+  $(document).ready(function () {
+    $('.not-registered').on('toggle', function () {
+      Webtrends.multiTrack({argsa: ['DCSext.GPAppointment', 'Not_Registered', 'WT.dl', '121']});
+    });
+  });
+});
```
Comment (st3v3nhunt) [inline public/js/custom-webtrends.js]: New line please

File: .env_sample [hunk 2]
```diff
@@ -0,0 +1,3 @@
+GOOGLE_ANALYTICS_TRACKING_ID=GA1
```
Comment (beseven) [inline .env_sample]: yes, without an .env file there will be some errors in the console, especially ga errors.
screen shot 2017-04-10 at 16 15 42

Do you want me to update the README?

Comment (st3v3nhunt) [inline .env_sample]: There shouldn't be any errors as the values will be defaulted to empty which should prevent the code branches where the includes are added from being executed. AFAIK the convention in the other repos is not to have any sample env files within the repo.
When specific things wish to be tested I'd probably look to either set the values on the command line or within the `docker-compose` files.

Comment (beseven) [inline .env_sample]: ga's values needs to start with an Integer, I'll make the change in the `docker-compose` default.

**Comment by c2s-dev** [comment]
:rocket: deployment of gp-finder-pr-78 in nhsuk-dev succeeded (http://gp-finder-pr-78.dev.c2s.nhschoices.net)

**Pull request merged**